botCommands/faq.test.js

```diff
@@ -0,0 +1,63 @@
+const command = require('./faq');
+const { generateMentions } = require('./mockData');
+
+describe('/faq', () => {
+  it.each([
+    ['/faq'],
+    [' /faq'],
+    ['/faq @odin-bot'],
+    [' /faq @odin-bot'],
+  ])("'%s' triggers the callback", (string) => {
+    expect(command.regex.test(string)).toBeTruthy();
+  });
+
+  it.each([
+    ['/fEhq'],
+    ['fye'],
+    ['/f'],
+    ['/fq'],
+    ['/f a q'],
+    ['```function("/faq", () => {}```'],
+    ['/FAQ'],
+    [''],
+    [' '],
+    ['/ '],
+    ['@odin-bot / faq'],
+    ['/fafaq'],
+    ['/bigNum'],
```

Comment:
Let's remove the `/bigNum` test case, we mostly want to test for edge cases at risk of triggering the regex

Reply:
On it